src/rpc/blockchain.cpp

```diff
@@ -1034,6 +1034,10 @@ UniValue mempoolInfoToJSON()
     ret.push_back(Pair("maxmempool", (int64_t)maxmempool));
     ret.push_back(Pair("mempoolminfee", ValueFromAmount(mempool.GetMinFee(maxmempool).GetFeePerK())));
 
+    char buf[20];
+    sprintf(buf, "%.2f", mempool.TransactionsPerSecond());
```

Comment:
I know that its likely impossible to trigger this bug, but by using a buffer that is smaller than the max possible %f, we technically have to prove that mempool.TransactionsPerSecond() cannot return a large number.  Instead let's use snprintf or at least 50 byte buffer:  https://stackoverflow.com/questions/7235456/what-are-the-maximum-numbers-of-characters-output-by-sprintf-when-outputting-flo

Reply:
As maybe a further nit, is the value meant primarily for human consumption, or is it meant to be further processed? After grepping through the `rpc/` directory, it looks like there's a couple values that are directly `json`-ized as `double`s but nothing that `sprintf`s `float`s or `double`s. 

So maybe it would be easiest to just cast to `double`?